docs/intro/overview.rst

```diff
@@ -46,14 +46,14 @@ http://quotes.toscrape.com, following the pagination::
 Put this in a text file, name it to something like ``quotes_spider.py``
 and run the spider using the :command:`runspider` command::
 
-    scrapy runspider quotes_spider.py -o quotes.json
+    scrapy runspider quotes_spider.py -o quotes.jl
 
 
-When this finishes you will have in the ``quotes.json`` file a list of the
+When this finishes you will have in the ``quotes.jl`` file a list of the
 quotes in JSON format, containing text and author, looking like this (reformatted
 here for better readability)::
 
-    [{
+    {
         "author": "Jane Austen",
         "text": "\u201cThe person, be it gentleman or lady, who has not pleasure in a good novel, must be intolerably stupid.\u201d"
     },
```

Comment:
should it have a comma here? Also, it should be a line per record, which may be important to understand in order to be able to parse the file

Reply:
Updated from a local run of the spider, to be safe.